.atomist/handlers/utils/GitHub.ts

```diff
@@ -0,0 +1,58 @@
+
+import { CommandPlan, CommandRespondable, Execute, Respond } from "@atomist/rug/operations/Handlers";
+
+const gh = "https://api.github.com";
+
+export const TOKEN_PATH = "github://user_token?scopes=repo";
+
+const headers = {
+    "Content-Type": "application/json",
+    "Authorization": `token #{${TOKEN_PATH}}`,
```

Comment:
Should probably have the `Accept: application/vnd.github.v3+json` header too.

https://developer.github.com/v3/#current-version